src/main/kotlin/com/terraformation/backend/customer/db/OrganizationStore.kt

```diff
@@ -148,25 +155,40 @@ class OrganizationStore(
             modifiedBy = currentUser().userId,
             modifiedTime = clock.instant(),
             name = row.name,
+            organizationTypeId = row.organizationTypeId,
+            organizationTypeDetails = row.organizationTypeDetails,
             timeZone = row.timeZone,
+            website = row.website,
         )
 
-    organizationsDao.insert(fullRow)
+    return dslContext.transactionResult { _ ->
```

Comment:
If the missing transaction scope was intentional, I can remove this.

Reply:
Not intentional; this is a good change.